### DISCUSSION TITLE
How to transform routes of a project

### DISCUSSION THREAD

**Discussion by wwachsmann** [spatie/laravel-typescript-transformer#40] [Q&A]
Hi guys,

i've seen a nice functionality about transforming the existing routes in a project on a video from Freek at the Laracon US 2023 Nashville (https://youtu.be/CrO_7Df1cBc?si=AbnkiTq2XMasGi3M).

But i did not find anything about this in the documentation on how it can be used or needs to be configured. Isn't it a part of the stable release? Or am i to stupid to find it?

Love to hear from you guys.

Thanks in advance and best regards
Werner

**Comment by rubenvanassche** [comment]
Hi @wwachsmann,

It is part of typescript transformer v3, which we hopefully release somewhere at the end of Q2  this year.
